mlrun/api/db/sqldb/models/models_mysql.py

```diff
@@ -522,6 +522,18 @@ class DataVersion(Base, mlrun.utils.db.BaseModel):
         def get_identifier_string(self) -> str:
             return f"{self.version}"
 
+    class DatastoreProfile(Base, mlrun.utils.db.BaseModel):
+        __tablename__ = "datastore_profiles"
+        __table_args__ = (
+            UniqueConstraint("name", "project", name="_datastore_profiles_uc"),
+        )
+
+        id = Column(Integer, primary_key=True)
+        name = Column(String(255, collation=SQLCollationUtil.collation()))
+        project = Column(String(255, collation=SQLCollationUtil.collation()))
+        type = Column(String(255, collation=SQLCollationUtil.collation()))
+        body = Column(String(1024, collation=SQLCollationUtil.collation()))
```

Comment:
The `body` may be a complex data-structure, right? You should to what we do in other objects - keep a `_full_object = Column("object", JSON)` column and provide accessors that wrap/unwrap it. This takes care of longer strings as well.